packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -2244,828 +2001,132 @@ function diffHydratedGenericElement(
-          domElement,
-          propKey,
-          'crossorigin',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'accentHeight':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'accent-height',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'alignmentBaseline':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'alignment-baseline',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'arabicForm':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'arabic-form',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'baselineShift':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'baseline-shift',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'capHeight':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'cap-height',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'clipPath':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'clip-path',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'clipRule':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'clip-rule',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'colorInterpolation':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'color-interpolation',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'colorInterpolationFilters':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'color-interpolation-filters',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'colorProfile':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'color-profile',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'colorRendering':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'color-rendering',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'dominantBaseline':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'dominant-baseline',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'enableBackground':
-        hydrateAttribute(
-          domElement,
-          propKey,
-          'enable-background',
-          value,
-          extraAttributes,
-        );
-        continue;
-      case 'fillOpacity':
+      case 'xHeight':
         hydrateAttribute(
           domElement,
           propKey,
-          'fill-opacity',
+          'x-height',
```

Comment:
did you mean to have x-height here?

Reply:
hmm? in ReactDOMComponent I see xHeight as the only call to hydrateAttribute() other than className tabIndex and the namespaced ones